public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -75,65 +102,193 @@ export const evaluateEveryValidationOptions: RegisterOptions = {
   },
 };
 
-export const GrafanaEvaluationBehavior = () => {
+function FolderGroupAndEvaluationInterval({
+  initialFolder,
+  initialRuleName,
+  initialEvaluateEvery,
+}: {
+  initialFolder: RuleForm | null;
+  initialRuleName: string;
+  initialEvaluateEvery: number;
+}) {
   const styles = useStyles2(getStyles);
-  const [showErrorHandling, setShowErrorHandling] = useState(false);
+  const [editInterval, setEditInterval] = useState(false);
   const {
+    setFocus,
     register,
     formState: { errors },
     watch,
+    setValue,
   } = useFormContext<RuleFormValues>();
 
-  const { exceedsLimit: exceedsGlobalEvaluationLimit } = checkEvaluationIntervalGlobalLimit(watch('evaluateEvery'));
-
+  const group = watch('group');
+  const folder = watch('folder');
   const evaluateEveryId = 'eval-every-input';
+  const evaluateEvery = watch('evaluateEvery');
+  const rulerRuleRequests = useUnifiedAlertingSelector((state) => state.rulerRules);
+  const groupfoldersForGrafana = rulerRuleRequests[GRAFANA_RULES_SOURCE_NAME];
+
+  const someRulesBelongToThisGroup = (
+    initialRuleName: string,
+    rulerRules: RulerRulesConfigDTO | null | undefined,
+    group: string,
+    folder_: string
+  ) => {
+    const folderObj: Array<RulerRuleGroupDTO<RulerRuleDTO>> = rulerRules ? rulerRules[folder_] : [];
+    const groupObj = folderObj.find((rule) => rule.name === group);
+
+    const rules = groupObj?.rules ?? [];
+    return rules.find((rule) => (isRulerGrafanaRuleDTO(rule) ? rule.grafana_alert.title !== initialRuleName : false));
+  };
+
+  const onBlur = () => setEditInterval(false);
+
+  useEffect(() => {
+    group &&
+      folder &&
+      setValue('evaluateEvery', getIntervalForGroup(groupfoldersForGrafana?.result, group, folder?.title ?? ''));
+  }, [group, folder, groupfoldersForGrafana?.result, setValue]);
+
+  useEffect(() => {
+    editInterval && setFocus('evaluateEvery');
+  }, [editInterval, setFocus]);
+
+  const intervalHasChanged = initialEvaluateEvery !== parseDurationToMilliseconds(evaluateEvery);
+  return (
+    <div>
+      <FolderAndGroup initialFolder={initialFolder} />
+      <Card className={styles.cardContainer}>
+        <Card.Heading>Group behaviour</Card.Heading>
+        <Card.Meta>
+          {`Evaluation interval applies to every rule within a group. 
+            It can overwrite the interval of an existing alert rule. 
+            Click on 'Edit group behaviour' button to edit this group value.`}
+        </Card.Meta>
+        <Card.Actions>
+          {intervalHasChanged &&
+          !editInterval &&
+          someRulesBelongToThisGroup(initialRuleName, groupfoldersForGrafana?.result, group, folder?.title ?? '') ? (
+            <>
+              <div className={styles.intervalChangedLabel}>
+                <Icon name="exclamation-triangle" size="xs" className={cx('text-warning', styles.warningIcon)} />
```

Comment:
Can we replace the instances of `text-warning` to a custom CSS rule with 

```css
color: ${theme.colors.warning.text};
```

it appears that class has some weird `:hover` states I don't think we need :)

Reply:
> It appears the group selection isn't cleared when another folder is selected – I'm thinking that's something we want to do?
I've just pushed a commit for fixing this 😊

Reply:
> The warning text about double checking the For duration seems to also show if the interval is the same as the existing one – a bug or should we just hide that?
Actually, I think we should remove this warning, as now we have a new validation in case this evaluation interval produces invalid For on another rules in the same group.